README.md

```diff
@@ -1,7 +1,7 @@
-CorrugatedIron - a .NET client for Riak
+RiakClient - a .NET client for Riak
```

Comment:
`RiakClient` -> `Riak .NET Client` - a client library for Riak for applications running on the .NET CLR.

Does the following seem OK?

* "Riak .NET Client" - the official name of the client when spaces and using the string ".NET" is OK, like in documentation or in the soon-to-be-created (or renamed) NuGet package title.
* `riak-dotnet-client` - the GH repo name and name of the project when it has to be part of a URL.
* `RiakClient` - the NuGet package name, and `.dll` name. Using [this package](http://www.nuget.org/packages/Microsoft.Net.Http/) as an example, the title is "Riak .NET Client" and installation command is `Install-Package RiakClient`. The other option is to follow the [Ruby gem](https://rubygems.org/gems/riak-client) scheme in which case the dll would be `riak-dotnet-client.dll` and NuGet package command `Install-Package riak-dotnet-client`